server/paths/refer.ts

```diff
@@ -0,0 +1,9 @@
+import { courseOfferingPath } from './find'
+
+const referralStartPath = courseOfferingPath.path('refer')
+
+export default {
+  referrals: {
```

Comment:
How would you feel about removing the top level of this, so we'd just have a `start` key. Are there any non-referral based paths we might have in here? Or were you thinking of matching the route structure here?

Reply:
I was more matching what we did on find - could we not remove the `courses` key over there by the same principle? Perhaps we should

Reply:
What structure did you use for the people show path?

Reply:
Yeah I think it might make sense